docs/developer/contributing.rst

```diff
@@ -197,30 +200,28 @@ contain private and/or sensitive information.
     ends up disabled. Stay prudent, and be extra cautious when working with
     sensitive information stored within a notebook in a Git repository.
 
-To install ``nbstripout`` run the following within the pymedphys directory:
+To install ``nbstripout``, run the following within the pymedphys directory:
 
 .. code:: bash
 
-    conda create --name pmp python=3.7 # Optional.
-    conda activate pmp # Optional
-    conda install nbstripout
-    nbstripout --install
-    nbstripout --status
+    λ conda create --name pmp python=3.7 # Optional.
+    λ conda activate pmp # Optional
+    λ conda install nbstripout
+    λ nbstripout --install
+    λ nbstripout --is-installed && echo Success!
```

Comment:
I think we should remove the `λ` here, what do you think?

Reply:
I see what you're doing now. Making sure success! is different from the commands.

Reply:
I actually put it in deliberately to distinguish between commands typed by the user (text after a lambda) and outputs from commands (text not after a lambda).

I do think if we remove the lambda, we need some way of distinguishing between the two text types. We don't want people typing "Success!" and wondering why it's not a recognized command...

Reply:
> I see what you're doing now. Making sure success! is different from the commands.
Yep! haha. Commented at the same time

Reply:
Potentially we can remove this section of the docs however... I've collapsed this installation into `yarn bootstrap` combined with a pre-commit hook that checks that `nbstripout` is installed. Let me know what you think.

Reply:
Agree. Let's remove it.

Reply:
Maybe instead of removing, moving it to another doc where we describe what is happening under the hood when `yarn conda:create` is run?